### PR TITLE
Changed search field to drop down list

### LINKED ISSUE #4: On FBI search page change Input field to Drop-down list
On FBI search page change Input field to Drop-down list with the following values:
- all
- title
- description
- details
- sex
- race
- id
- hair color
- weight
- classification
- url

### PR DESCRIPTION
**Pull request by olegkolyaka78**
Changed search field to drop down list on FBI, Twitter and Newsdata search pages

### CODE REVIEW

File: 1 [hunk 1]
```diff
@@ -0,0 +1,35 @@
+[33mcommit de00ad8581ec7c0b0ae7f768b9e5c4d51af3f7aa[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;31morigin/master[m[33m, [m[1;31morigin/HEAD[m[33m)[m
```
Comment (mechamanny) [inline 1]: I think that this file is not a part of this feature, would you mind deleting it?

File: Gemfile.lock [hunk 2]
```diff
@@ -103,7 +103,7 @@ GEM
     minitest (5.15.0)
     msgpack (1.4.5)
     nio4r (2.5.8)
-    nokogiri (1.13.1-x86_64-darwin)
+    nokogiri (1.13.1-x86_64-linux)
```
Comment (mechamanny) [inline Gemfile.lock]: I have a feeling that we're going to be flip-flopping this dependency based on everyones environment. I think we should either not add this change, or explore a common dev env using something like docker to reduce this kind of noise.

File: app/views/ajax/newsdata.html.erb [hunk 3]
```diff
@@ -6,10 +6,17 @@
...
 
 <div id='crud_operations_div'>
-  <b><label>Valid options:  all, title, link, description</label></b>
-  <input id="newsdata_api_word">
-
-  <br>
+  <div class="field">
+  <b><label for="newsdata_api_word">Choose a Search Option :</label></b>
+
+    <select name="newsdata_api_word" id="newsdata_api_word">
```
Comment (mechamanny) [inline app/views/ajax/newsdata.html.erb:12]: Check out the rails helper for creating select fields, I think it makes sense to use rails library methods where possible.

https://guides.rubyonrails.org/v2.3/form_helpers.html#making-select-boxes-with-ease

File: 1 (hunk 1 above)
Comment (olegkolyaka78) [reply 1]: I don't mind, even don't remember how this file was created.

File: Gemfile.lock (hunk 2 above)
Comment (olegkolyaka78) [reply Gemfile.lock]: Was it me who changed version? How could it happened?

Comment (jimenezmiguela) [reply Gemfile.lock]: Changed to nokogiri (1.13.1-x86_64-darwin).  Any questions please ask.

**Review by jimenezmiguela — APPROVED** [review]

**Pull request merged**